templates/events/details.html

```diff
@@ -125,6 +125,18 @@ <h2 id="event-details-heading">
                                         </div>
                                     </div>
                                 {% endif %}
+                                {% if payment %}
+                                    <div class="row">
+                                        <div class="event-hero-meta-info">
+                                            <div class="col-xs-12 col-sm-4 col-md-4 col-md-offset-4 col-sm-offset-4">
+                                                <div class="event-hero-title">Pris</div>
+                                                <div class="event-hero-large-text">
+                                                    {{  payment.price }};-
```

Comment:
Should be `,-`, not `;-`. If you already have the price with KR and ØRE in the model, you don't need to end the text with `,-` at all (`100,00 kr` is fine, you don't need `100,00,- kr`)